resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -218,3 +218,218 @@
       re-save-file-list fastqc_archive2 "${RW_FASTQC[@]}"
       re-save-list fastqc_url "${FW_FASTQC_URL[@]}"
       re-save-list fastqc_url2 "${RW_FASTQC_URL[@]}"
+
+- slug: files-to-fastq-single
+  name: Convert files to Reads (single-end)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq
+  data_name: 'Files to FASTQ single-end ({{ src.0.file.file }})'
+  version: 0.0.1
+  type: data:reads:fastq:single
+  flow_collection: sample
+  category: upload
+  persistence: RAW
+  description: >
+    Convert FASTQ files to single-end reads.
+  input:
+    - name: src
+      label: NGS reads (FASTQ)
+      type: list:data:file
+      description: >
+        NGS reads in FASTQ format.
+  output:
+    - name: fastq
+      label: Reads file
+      type: list:basic:file
+    - name: fastqc_url
+      label: Quality control with FastQC
+      type: list:basic:file:html
+    - name: fastqc_archive
+      label: Download FastQC archive
+      type: list:basic:file
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      {% for read in src %}
+        cp {{ read.file.file }} .
+      {% endfor %}
+
+      FASTQGZ=(*.fastq.gz)
```

Comment:
What happens if the input files are uncompressed (.fastq instead of .fastq.gz)?

Reply:
It won't work with uncompressed ones. Do we need to support `.fastq` also?

Reply:
Let's not do this at this point.